### DISCUSSION TITLE
Down and Offline red box should not be mixed

### DISCUSSION THREAD

**Discussion by zilvernet** [netalertx/NetAlertX#622] [General]
Hi, I noticed on the latest version: v24.3.19

That on the devices the down box now includes also offline.

<img width="214" alt="Screenshot 2024-04-12 at 9 41 37 AM" src="https://github.com/jokob-sk/NetAlertX/assets/75147745/c541aed4-7a1d-46da-bc2c-45e031aa5eda">

I think is a mistake to include offline items that are not part of a monitoring check:

<img width="175" alt="Screenshot 2024-04-12 at 9 44 19 AM" src="https://github.com/jokob-sk/NetAlertX/assets/75147745/5f776dc7-ddb7-4d94-9ea1-18923de4de47">

<img width="1235" alt="Screenshot 2024-04-12 at 9 46 59 AM" src="https://github.com/jokob-sk/NetAlertX/assets/75147745/19b9a81a-6411-481a-8920-08b097a20afe">

It should be easier for an admin to view what essential equipment are offline.

If you would like to include offline filters specifically it should have its independent Dashboard box.

**Comment by jokob-sk** [comment]
Hi, as a compromise I added a left-hand menu item for filtering down devices:

![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/f08f867d-1e74-43f3-ab38-531d8e7d3f09)

I want to keep the tile count at 6 so the mobile view is more manageable. Hope this works.

**Comment by zilvernet** [comment]
I think the main dashboard red box should not alert for offilines or at least there should be a configuration setting so that users can tweak this filter.

To me this change represents a big issue, since i monitor the main page for down (monitored devices) and when i have no reds it means that my network critical devices is working properly.  (and if i see a red item i can focus on the device that is damaged)

Generally speaking on any given network there will always be offline devices like mobile phones, etc. so the offline number always tend to increase over time making the red dashboard box that now combines the count for down and offline useless because it will always show a red count for a device .  Even if you archive the offline they will still show on the down & offline dashboard box, leaving the only option to clear these red items by manually deleting all the offline devices.

In the mean time I will be running the previous version that didn't have this new feature.
I hope you can reconsider this change.

**Reply by jokob-sk** [reply]
Just FYI this has been rolled back, so Down and Offline devices are separated.

**Comment by Byrnesdigital** [comment]
Just a curiosity, what is the difference between devices that are "down" vs devices that are "offline"? I have 0 devices under Down and a few (cellphones, turned off laptop, etc) under offline. 
![image](https://github.com/user-attachments/assets/eb5f6e88-0fd9-4d53-a6d7-fad5cc287aaa)

**Reply by jokob-sk** [reply]
Hi @Byrnesdigital ,

Thanks for asking!

It's devices where you marked them as "Alert Down". You can find more info here: https://github.com/jokob-sk/NetAlertX/blob/main/docs/NOTIFICATIONS.md#device-settings-

Let me know if you have a suggestion to improve the docs or descriptions!

**Comment by AlexRJ-FR** [comment]
Thank you for your answer to understand the difference between "Down" and "Offline".
- Is there a way to send a "Wake on LAN" when the device is "Down"
- Is there a way to send a "Wake on LAN" when the device is "Offline"

**Comment by jokob-sk** [comment]
Not currently. Open a feature request and I might look into it.
